### ISSUE TITLE
SCAYT doesn't seem to work on IE10

### ISSUE THREAD

**Issue by jswiderski** [WebSpellChecker/ckeditor-plugin-scayt#156]
1. Load CKEditor with SCAYT (we have also checked the pre-release 4.9.0 where we have upgraded the SCAYT - https://github.com/ckeditor/ckeditor-presets/commit/8ad0a817e6365ed61e6f389579774d5e4cda6bca) 
2. Enable SCAYT and start typing.
Result:
```
SCRIPT5007: Invalid argument for „in”: Object expected
ckscayt.js, Line 34 
```

Does SCAYT still support older versions of IE?

**Comment by jswiderski** [comment]
Could be related to https://github.com/WebSpellChecker/ckeditor-plugin-scayt/issues/155.

Guys, does SCAYT support IE8-10?

**Comment by wsc-bug-hunter** [comment]
Hello @jswiderski Currently we are working on this issue. It is related to #155.

**Comment by wsc-julia-shaptala** [comment]
Hi @jswiderski,

By default, SCAYT is supported for IE9+ and MS Edge, and compatible with IE8. However, we have encountered the problem with IEs and at the moment we are trying to fix it. The fix will be definitely included in our upcoming release 5.00. It is scheduled for the second week of March.

The problem is in SCAYT core and not is related to SCAYT plugin itself. 

At the moment we haven't received any requests from end customers about the problem with older IEs. It looks like that IE8-IE10 are not so often used. However, in any case, this will be fixed.

Is the issue with IE compatibility blocker for you?

Thank you,

**Comment by wsc-julia-shaptala** [comment]
This has been fixed as a part of WSC 5.0.0 release on May 16, 2018.
